tests/test_squad.py

```diff
@@ -0,0 +1,295 @@
+# Copyright (c) 2019 NVIDIA Corporation
+import unittest
+import os
+import sys
+import json
+
+import shutil
+
+from .context import nemo, nemo_nlp
+from .common_setup import NeMoUnitTest
+from nemo.utils.lr_policies import get_lr_policy
+from nemo_nlp.utils.callbacks.squad import \
+    eval_iter_callback, eval_epochs_done_callback
+
+
+class TestSquad(NeMoUnitTest):
+
+    @classmethod
+    def setUpClass(cls) -> None:
+        super().setUpClass()
+        data_folder = os.path.abspath(os.path.join(os.path.dirname(__file__),
+                                                   'data/nlp'))
+        if not os.path.exists(data_folder):
+            print(f"mkdir {data_folder}")
+            os.mkdir(data_folder)
+
+        squad_folder = data_folder + '/squad'
+        if not os.path.exists(squad_folder):
+            download_script_path = os.path.abspath(
+                os.path.join(os.path.dirname(__file__),
+                             '../examples/nlp/scripts'))
+            sys.path.insert(0, download_script_path)
+            from download_squad import SquadDownloader
+            print("Extracting Squad data to: {0}".format(squad_folder))
+            squad_dl = SquadDownloader(data_folder)
+            squad_dl.download()
+
+            squad_v1_dev_file = \
```

Comment:
How big is this dataset? If it is big, maybe we should move it to CI instead of unittests.

Reply:
its only part of the dataset. Both tests take 100s